<launch>
  <arg name="debug_directory" value="$(env HELLO_FLEET_PATH)/debug/"/>
  <node name="object_manager" pkg="rail_stretch_perception" type="object_manager.py"/>
  <node name="object_detector" pkg="stretch_deep_perception" type="detect_objects_python3.py">
    <param name="debug_directory" type="string" value="$(arg debug_directory)"/>
  </node>
</launch>